sound/soc/sof/sof-of-dev.c

```diff
@@ -103,6 +111,7 @@ static int sof_of_remove(struct platform_device *pdev)
 static const struct of_device_id sof_of_ids[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_IMX8)
 	{ .compatible = "fsl,imx8qxp-dsp", .data = &sof_of_imx8qxp_desc},
+	{ .compatible = "fsl,imx8qm-dsp", .data = &sof_of_imx8qm_desc},
 #endif
```

Comment:
Need to also add documentation bindings as a separate patch, even if that patch won't go through Mark's asoc tree.

Reply:
Added and pushed, waiting for GH to update PR. You can check the branch directly in the meanwhile.

Reply:
Thanks, looks good to me.